convert_tapping_dataset_to_nirs.py

```diff
@@ -1,55 +1,139 @@
+import os
 import sys
-import re
+from collections import OrderedDict
 from pathlib import Path
 
+import click
 from pynwb import NWBHDF5IO
 
-# todo compile conversion tools into an installable package
-sys.path.append("src")
-from snirf import load_snirf, load_stim_table
+# Ensure modules in `src` are findable before importing
+sys.path.append(os.path.join(os.path.dirname(__file__), "src"))
+from bids import (
+    get_bids_version,
+    get_coordinate_frame_description,
+    get_dataset_authors,
+    get_sidecar_metadata,
+    get_subject_dir,
+    list_subject_ids,
+    load_stim_table,
+)
 from nwb import convert_to_nwb
+from snirf import get_format_version, load_snirf
+
+CONVERTER_VERSION = "0.1.0-dev.001"
+
+# copied from: https://github.com/rob-luke/BIDS-NIRS-Tapping/blob/master/README.md
+EXPERIMENT_DESCRIPTION = """This experiment examines how the motor cortex is activated
+during a finger tapping task. Participants are asked to either tap their left thumb to
+fingers, tap their right thumb to fingers, or nothing (control). Tapping lasts for 5
+seconds as is propted by an auditory cue. Sensors are placed over the motor cortex as
+described in the montage section in the link below, short channels are attached to the
+scalp too. Further details about the experiment (including presentation code) can be
+found at https://github.com/rob-luke/experiment-fNIRS-tapping.
+"""
+INSTITUTION = "Macquarie University"
+PUBLICATIONS = ""
+KEYWORDS = ["fNIRS", "Haemodynamics", "Motor Cortex", "Finger Tapping Task"]
+
+
+@click.command()
+@click.argument(
+    "dataset_path", type=click.Path(exists=True, file_okay=False, path_type=Path)
+)
+def main(dataset_path):
+    """Convert the fNIRS finger-tapping dataset to NWB for upload to the DANDI Archive.
```

Comment:
```suggestion
    """Convert the fNIRS finger-tapping dataset to NWB:N for upload to the DANDI Archive.
```
Is the NWB format technically called NWB:N? I don't think the NWB 1.0 format is (widely) used, so maybe this is just being pedantic 🙂

Reply:
I tried to look up in the documentation to see what is the standard name (NWB:N or NWB). Even the documentation seems to be confused ([example](https://pynwb.readthedocs.io/en/stable/overview_nwbfile.html)).

Before going through and modifying all references to NWB in this repo - do you know where there is a reference that clarifies which is the current terminology?

Reply:
That example is pretty egregious inconsistency haha. I can't find any convention that would explain why they use NWB vs. NWB:N at different times. I'm fine with keeping everything `NWB`.

Reply:
:laughing: ok, sounds good.